README.md

```diff
@@ -8,99 +8,166 @@ _user_ not a _developer_.
+
+Be patient with yourself as you learn this material.
+
+### Terminal Programs
+
+A terminal program is a program that listens for keyboard input and hands it
+off to a magical genie who lives inside your computer. Also, when the magical
+genie tells the terminal to draw letters on its screen, it does so. For
+example, when you are in a terminal program and type the letter `l` the
+terminal program senses the `l` and tells the genie. The genie then tells the
+terminal to draw an `l` on its screen. The terminal then looks up your font
+preferences and color preferences and draws the appropriate `l`.
+
+This might be confusing, but let's compare it to television. A television is a
+device that listens to your remote control and then tells a magical genie
+(different genie, of course) which feed of data to find. When the genie finds
+the data (maybe on a coaxial cable, or satellite dish) it commands the
+television to display the picture. On most modern televisions, in fact, the
+magical genie tells the television to draw a picture 120 times per second!
+
+The essential fact is that a terminal is a means for talking to the operating
+system's genie.  In lessons we'll refer to the "terminal" as the thing you type
+in, as a program you can launch, as a space you can click in.
+
+## Recognize a Shell Interface
+
+### Shell Interfaces Described
+
+OK, sorry, genies don't exist. Well, at the very least, operating system genies
+don't exist.
+
+Above, when we said that the terminal hands requests and receives commands from
+a genie, a thing that _thinks_ and thing that _can find out things_, we were
+actually talking about "the shell." The _shell_ knows how to ask the operating
+system how big the hard drive is. The _shell_ knows how to ask the operating
+system if the web site `flatironschool.com` can be reached. The _shell_ **does
+not** know how to display "Hello World" in 16 point Monaco font on a black
+background, that's the _terminal_.
+
+A shell is a program hosted _inside of a terminal_ that interacts with the
+operating system on your behalf. It's because of the fact that you can't
+interact with a _shell_ without a _terminal_ that some imprecisely call the
+_terminal_ "the shell."
+
+The shell, since it can think and do things, takes input that you type in. It
+looks at that input and asks the operating system to interpret that request.
+When the operating system comes back with results, the shell then hands back to
+the terminal (for displaying) the result of the command.
+
+When you see a terminal, the shell hosted inside of it it will present to you a
+"command prompt." This means, from the computer's perspective "I'm
+listening..." Prompts may look different from computer to computer. Customized
+prompts express developer uniqueness. We've seen weather reports, beautiful
+colors, computer temperatures in prompts. At heart, though, they they all have
+the same function: to carry your command to the operating system and run it.
+
```

Comment:
Might be cool to have a couple more examples of the command prompt? Also probably a good thing to have a shot of the in browser IDE prompt so they know exactly what you're talking about. One of my friends built this tool to customize your BASH profile, which might just confuse the issue: https://xta.github.io/HalloweenBash/